packages/orchestrator-ui-components/src/pages/startPage/WfoStartPage.tsx

```diff
@@ -217,30 +247,59 @@ const outOfSyncSubscriptionsListSummaryQueryVariables: GraphqlQueryVariables<Sub
         ],
     };
 
-const processListSummaryQueryVariables: GraphqlQueryVariables<Process> = {
-    first: 5,
-    after: 0,
-    sortBy: {
-        field: 'startedAt',
-        order: SortOrder.DESC,
-    },
-    filterBy: [
-        {
-            // Todo: isTask is not a key of Process
-            // However, backend still supports it. Field should not be a keyof ProcessListItem (or process)
-            // https://github.com/workfloworchestrator/orchestrator-ui/issues/290
-            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-            // @ts-ignore waiting for fix in backend
-            field: 'isTask',
-            value: 'false',
-        },
-        {
-            field: 'lastStatus',
-            value: 'created-running-suspended-waiting-failed-resumed-inconsistent_data-api_unavailable-awaiting_callback',
+const getMyWorkflowListSummaryQueryVariables = (
+    username: string,
+): GraphqlQueryVariables<Process> => {
+    return {
+        first: 5,
```

Comment:
move some fields in a default object, to use in each `...QueryVariables` and move isTask filter to its own variable so you only add the Todo once.
```
return {
    ...defaultQueryVars,
    filterBy: [filterByNotTask, ...]
}
```